.docker/WebUI/Configuration/Configuration.jsx

```diff
@@ -2,11 +2,10 @@
 
     export default {
         // Arriba Service URL to query
-        // When using NGINX as Reverse Proxy it can be setup as the frontend url plus /data
-        url: window.location.protocol + "//" + 
-             window.location.hostname + 
-             (window.location.port > 0 ? ":" + window.location.port : "" ) + 
-             "/v2",
+        // When using NGINX as Reverse Proxy it can be setup as the frontend url plus /v2 or v1 
+        url: window.location.protocol + "//" + window.location.host + "/v2",
+        
+        oauthUrl: window.location.protocol + "//" + window.location.host + "/oauth/",
```

Comment:
I like this change, but really don't like the duplication

Reply:
I tried to use a baseUrl property and reuse at the next line, but it didn't work. I will look to a different way to do that because it was bordering me as well

Reply:
I had to create a variable before the export.

```
    let baseArribaUrl = window.location.protocol + "//" + window.location.host;
```

then at the export

```
url:  baseArribaUrl + "/v2",
        
oauthUrl: baseArribaUrl + "/oauth/",

```